org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_detail.scss

```diff
@@ -65,11 +65,28 @@
 
   .crm-summary-block-wrap {
     width: 100%;
+    margin-top: 0;
   }
 
   #crm-contactname-content {
     border: none !important;
+    height: 60px;
```

Comment:
Magic number (as in, a very specific number that seems ad-hox). What's the reason for this? Doesn't forcing the height of this block create problems with the dynamic content?